components/table/table-wrapper.js

```diff
@@ -54,10 +87,47 @@ export const tableStyles = css`
 		font-size: 0.7rem;
 		line-height: 0.9rem;
 	}
-	d2l-table-wrapper[type="default"] .d2l-table > thead > tr > th,
-	d2l-table-wrapper[type="default"] .d2l-table > * > tr.d2l-table-header > th,
-	d2l-table-wrapper[type="default"] .d2l-table > * > tr[header] > th {
-		height: 27px; /* min-height to be 48px including border */
+
+	/* sortable header cells */
+	d2l-table-col-sort-button {
+		vertical-align: middle;
+	}
+	d2l-table-col-sort-button:not(:only-child) {
+		--d2l-table-col-sort-button-additional-padding-inline-end: 0px; /* stylelint-disable-line length-zero-no-unit */
+	}
+	d2l-table-col-sort-button:not(:last-child) {
+		--d2l-table-col-sort-button-margin-inline-end: 1px;
+	}
+
+	/* TODO: once we only support browsers that support :has the section below can be removed up until @supports */
+	.d2l-table th.d2l-table-header-col-sortable {
+		height: var(--d2l-table-cell-overall-height);
+		padding: 0;
+	}
+	.d2l-table th.d2l-table-header-col-sortable-siblings d2l-table-col-sort-button {
+		--d2l-table-col-sort-button-width: unset;
+	}
+	.d2l-table-header-col-sortable-no-sorted.d2l-table-header-col-sortable-siblings :last-child {
+		padding-inline-end: calc(0.6rem + 18px);
+	}
+	.d2l-table-header-col-sortable :not(d2l-table-col-sort-button) {
+		margin: var(--d2l-table-cell-col-sort-button-size-offset);
```

Comment:
This adds a margin to non-table-col-sort-buttons that are in a cell with one. In our demos it looks weird without this, however consumers can be setting their own spacing on their cell contents so I don't know that it's right for us to be dictating that. Let me know any thoughts. I'm open to removing it from here and moving it elsewhere for the demo.

Reply:
Yeah this seems a bit invasive to me, and likely best handled by consumer code wherever it comes up.